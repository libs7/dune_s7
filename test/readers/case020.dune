(foo "line with embedded\nnewline")

;; end-of-line strings
(eolstring
 "\| this is an eol string
)
