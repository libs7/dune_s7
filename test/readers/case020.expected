((foo "line with embedded
newline") (eolstring " this is an eol string"))
